service_management/azure/test/integration/cloud_service/cloud_create_test.rb

```diff
@@ -15,29 +15,30 @@
 require 'integration/test_helper'
 
 describe Azure::CloudServiceManagementService do
-  include Azure::Core::Utility
-
   subject { Azure::CloudServiceManagementService.new }
+  let(:cloud_service_name) { 'testcloudservice2375' }
   let(:options) do
     {
-      location: 'West US',
-      description: 'Test'
+      location: 'East US',
```

Comment:
The test was failing that the location was not supported so i updated the location.